src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Button from "../../inline/Button";
+import playIcon from "../VideoHeader/PlayIcon.svg";
```

Comment:
Please don't import images from other components like this. This makes the import hierarchy asymmetrical, since it's only by coincidence that the VideoSpotlightBlock imports from the VideoHeader's image, even though the opposite would also be possible. Instead, as a general rule of thumb, if you find yourself doing that, you should extract the common part into a third area that they both depend on.

But this entire comment is moot, since it looks like the play icon in the design is actually much larger for this component than the Video Header component's. Could you make sure you use the properly sized icon?

Reply:
ok